tests/component/support/component.js

```diff
@@ -0,0 +1,44 @@
+// ***********************************************************
+// This example support/component.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands';
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
+import { h } from 'vue';
+import { mount } from 'cypress/vue';
+import i18n from '@/i18n';
+import vuetify from '@/plugins/vuetify';
+import { VApp, VMain } from 'vuetify/components';
+
+
+Cypress.Commands.add('mount', (component, options = {}) => {
+  return mount(
+    h(VApp, [
+      h(VMain, [
+        h(component, { ...options.props }) 
+      ]),
```

Comment:
This section is to try to get the dialog to be centered like it is when it's displayed in Prod, but it's not working... any insight @seriouslysean  @itsalaidbacklife 
<img width="1380" alt="Screenshot 2025-02-02 at 8 20 43 AM" src="https://github.com/user-attachments/assets/10ed5a12-6bd8-4751-9e29-bce0eff19e93" />